webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -79,7 +79,7 @@ def timestamp_to_date(timestamp, fmt="%Y-%m-%d %H:%M:%S"):
 
 
 @library.filter
-def time_tag(dt, format="%a, %b %d %H:%M %Z", future=False):
+def time_tag(dt, format="%a, %b %d, %Y at %H:%M %Z", future=False):
```

Comment:
This isn't really related to this PR, but I bumped into it when fiddling with things. Not having the year is really irritating for me. Socorro shows things over a 6 month period which can span years.